chapters/drawing.txt

```diff
@@ -776,10 +776,9 @@ The assembled primitives execute the bound graphics pipeline.
     If the sname:VkPipeline object bound to
     ename:VK_PIPELINE_BIND_POINT_GRAPHICS requires any dynamic state, that
     state must: have been set on the current command buffer
-  * [[VUID-vkCmdDrawIndexed-indexSize-00463]]
```

Comment:
Should I keep this VUID? It contains the `indexSize` name which is not a API thing.

Reply:
It's harmless to leave so long as the VU statement itself hasn't changed. The script just picks off the first pname: and uses it in the VUID, which is obviously not 100% reliable. Sometimes it can't find one and uses 'None', which is also OK.

Reply:
OK, then. The VU statement did not change meaning, only markup.